scripts/ci-build-azure-ccm.sh

```diff
@@ -90,13 +90,14 @@ can_reuse_artifacts() {
     echo "true"
 }
 
-cleanup() {
+capz::ci-build-azure-ccm::cleanup() {
+    echo "cloud-provider-azure cleanup"
     if [[ -d "${AZURE_CLOUD_PROVIDER_ROOT:-}" ]]; then
         make -C "${AZURE_CLOUD_PROVIDER_ROOT}" clean || true
     fi
 }
 
-trap cleanup EXIT
+trap capz::ci-build-azure-ccm::cleanup EXIT
```

Comment:
Good find! While you're here, should we rename all the cleanup functions called in ci scripts? I see a few other occurrences untouched

Reply:
ok, let me update all of the scripts!

Reply:
I think i got all of the occurrences - PTAL